src/graph/entities/edge.h

```diff
@@ -14,30 +14,16 @@ typedef int RelationID;
 
 #define EDGE_LENGTH_INF UINT_MAX - 2
 
-// instantiate a new unpopulated edge
-#define GE_NEW_EDGE()                 \
-(Edge) {                              \
-	.entity = NULL,                   \
-	.id = INVALID_ENTITY_ID,          \
-	.relationship = NULL,             \
-	.relationID = GRAPH_NO_RELATION,  \
-	.src = NULL,                      \
-	.dest = NULL,                     \
-	.srcNodeID = INVALID_ENTITY_ID,   \
-	.destNodeID = INVALID_ENTITY_ID   \
-}
```

Comment:
Why was this macro removed, don't we need a convenient way to create an initialized stack based edge ?